packages/appstore/src/components/modals/open-positions-svg-modal/open-positions-svg-modal.tsx

```diff
@@ -62,15 +47,29 @@ const OpenPositionsSVGModal = ({
                     )}
                 </Text>
                 <Text as='p' color='prominent ' size='xs'>
-                    {is_migrated_with_position ? (
+                    {!is_migrated_with_position ? (
```

Comment:
is this `negation` intentional? or something left after testing?

Reply:
ahh  yea.. was testing the implementation.. will remove it.. Good catch